telemetry/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+# docker compose for redis and grafana 
+# to add to redis image: 
+  # git integration + auto repo pull   
+  # usb port support within redis container
+
+version: '3.4'
+
+services:
+  redis:
+    container_name: redistimeseries
+    # image: redislabs/redistimeseries:latest # setting prebuilt image
+    build: 
+      context: .
+      dockerfile: redis_ts.dockerfile
+    ports:
+      - '6379:6379'
+    devices:
+      - "/dev/ttyUSB0:/dev/ttyUSB0"
```

Comment:
Also, not sure if this was just for me, but my devices usually end up at `/dev/ttyACM0`, so I had to change this to
```yaml
- "/dev/ttyACM0:/dev/ttyUSB0"
```
Worth looking into the difference, and seeing if there's a way for a USB device to go into a specific port or not...